etc/antora/supplemental-ui/partials/header-content.hbs

```diff
@@ -0,0 +1,33 @@
+<header class="header">
```

Comment:
Override the header of the default UI. We can remove it if we don't plan to host the documentation on GitHub Pages since https://neo4j.com/labs has already a custom site UI.